app/css/Form.scss

```diff
@@ -4,3 +4,12 @@
 .is-invalid-input {
   border-color: $danger;
 }
+
+@keyframes spinner {
```

Comment:
Do you think we could create a: TransactionForm component, that handle all the cases about loading/error/success
That way this scss would not be local but on the component itself?

Reply:
Ok yeah, I'll try